src/figuresClasses.ts

```diff
@@ -1,11 +1,83 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape: Figure['shape'] = 'triangle';
+
+  constructor(
+    public color: Figure['color'],
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const numbers: number[] = Array.from([a, b, c]).sort(
```

Comment:
The `Array.from([a, b, c])` should use `this.a`, `this.b`, and `this.c` instead of `a`, `b`, and `c` since the constructor parameters are automatically assigned to the class properties.

Reply:
Done